src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
+=====================
+
+..
+  gitlab.com > create account
+
+Project Members
+===================
+
+In the local development you must update the git files of each project to the new Gitlab repository.
```

Comment:
not clear, phrase it better. Something like update local working copy git configuration.